app/src/main/res/layout/dialog_results.xml

```diff
@@ -343,6 +343,7 @@
 
         <include layout="@layout/expanded_results_box"
             android:id="@+id/expandedResultsBox"
+            android:visibility="gone"
```

Comment:
Current implementation of arrow will display an emoji in textview... can we swap them out to 25B4 and 25B8 instead? Smaller triangle but none of them will turn into emojis.